drivers/hmis/app/graphql/types/table_column_config_type.rb

```diff
@@ -0,0 +1,16 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class TableColumnConfigType < Types::BaseEnum
+    graphql_name 'TableColumnConfigType'
+
+    value 'STRING', value: 'string'
+    value 'DATE', value: 'date'
```

Comment:
maybe add comment that this should align with `Hmis::TableConfiguration::COLUMN_TYPES` or even reuse that constant here